docs/remote_inference_blueprints/cohere_connector_chat_blueprint.md

```diff
@@ -0,0 +1,172 @@
+### Cohere Chat Connector Blueprint:
+
+This blueprint will show you how to connect a Cohere chat model to your Opensearch cluster. You will require a Cohere API key to create a connector.
+
+It is suggested to use the default `command` model for using Chat. Cohere's Chat endpoint also features Retrievel Augmented Generation (or RAG) parameters, by adding `connectors` or `documents` in your request body, in addition to allowing you to pass a `conversation_id` to provide context to Cohere's model. 
```

Comment:
> by adding `connectors` or `documents` in your request body

I think you mean Cohere connectors here, right?

Reply:
Yes correct, Cohere connectors - there should be text earlier linking docs and explaining the difference, but I can clarify that here as well